Composer/ScriptHandler.php

```diff
@@ -428,8 +434,8 @@ protected static function getPhpArguments()
     /**
      * Returns a relative path to the directory that contains the `console` command.
      *
-     * @param Event $event      The command event.
-     * @param string       $actionName The name of the action
+     * @param Event  $event      The command event.
+     * @param string $actionName The name of the action
```

Comment:
These two lines of cs fix are now merged in master, can you please revert them and rebase? Thanks :)